packages/hooks/src/useNFTContract.ts

```diff
@@ -60,3 +78,60 @@ export const useNFTMethod = (
         error: error,
     }
 }
+
+/**
+ * Hook to execute an NFT contract view method without an account
+ *
+ * @param {string} contractAddress - contract adress
+ * @param {string} methodName - the view method to execute
+ * @param {object} params - method parameters
+ * @returns {object} { error, loading, data }
+ */
+
+export const useViewNFTMethod = (
+    contractAddress: string,
+    methodName: string,
+    params: {}
+) => {
+    const config = { networkId, nodeUrl }
+
+    const fetcher = async (methodName: string, serializedParams: string) => {
+        const near = await connect({
+            networkId,
+            nodeUrl,
+            deps: {
+                keyStore: null,
+            },
+        })
+        const account = await near.account(null)
+
+        const contract = await new Contract(account, contractAddress, {
+            viewMethods: [
```

Comment:
Some view methods in `getContractMethods('nft')` are in the wrong place, and we can't use 
them without an account. I didn't want to change it for now so nothing break 
https://github.com/curaOS/source/blob/650e8c718a3d87686c5efffd18cdd764fba5eed2/packages/hooks/src/near-utils.ts#L40

All `useNFTMethod` and `useNFTViewMethod` and `useNFTContract` needs some tweaks and tests to prevent repetitive code, and also if we can make some way to pass custom config would be nice like [this one](https://github.com/ourzora/nft-hooks#configuration). This could be a new issue